test/integration/authorization_test.go

```diff
@@ -276,23 +277,37 @@ type subjectAccessReviewTest struct {
 }
 
 func (test subjectAccessReviewTest) run(t *testing.T) {
-	actualResponse, err := test.clientInterface.Create(test.review)
-	if len(test.err) > 0 {
-		if err == nil {
-			t.Errorf("%s: Expected error: %v", test.description, test.err)
-		} else if !strings.HasPrefix(err.Error(), test.err) {
-			t.Errorf("%s: expected\n\t%v\ngot\n\t%v", test.description, test.err, err)
+	failMessage := ""
+	err := wait.Poll(testutil.PolicyCachePollInterval, testutil.PolicyCachePollTimeout, func() (bool, error) {
+		actualResponse, err := test.clientInterface.Create(test.review)
+		if len(test.err) > 0 {
+			if err == nil {
+				return false, fmt.Errorf("%s: Expected error: %v", test.description, test.err)
```

Comment:
> It does not change the cases where you were expecting to be denied access to the SAR altogether, those already worked fine.

Depends if the thing we were checking was being granted the ability to run SAR's. Might not be applicable, I wasn't sure all the ways we were using subjectAccessReviewTest